schematic/models/commands.py

```diff
@@ -79,8 +79,8 @@ def model(ctx, config):  # use as `schematic model ...`
 @click.option(
     "--manifest_record_type",
     "-mrt",
-    default='both',
-    type=click.Choice(['table', 'entity', 'both'], case_sensitive=True),
+    default='combo',
+    type=click.Choice(['table', 'file', 'file_w_entities', 'combo'], case_sensitive=True),
```

Comment:
Might be worth having similar options: `file`, `file_w_entity`, `file_w_table`, `file_w_both`
Also we should consider renaming `table` to `DB` or something referencing a database in the future to match naming in synapse store and future asset store refactor and by synapse agnostic

Reply:
I like the options you laid out, making it more clear that files are always present.
I very much dislike `both` and `combo`...what if it was like: `file`, `file_and_entites`, `file_and_table`, `file_table_and_entities`. Or do we always assume file...then have `manifest_record_addition` instead of `manifest_record_type` and just accept `entities`, `table`, `entities_and_table`? @milen-sage

Reply:
I'd prefer the 2nd option, that seems more similar to our old convention of delineating only what is added in addition to the file. It could then be `entities`, `table`, `entities_and_table`, and then something to the effect of `None` for the file only upload. 
But if we prefer the first option then `file_table_and_entities` is better than `both`! I feel being more explicit is better in that case.

Reply:
I'll wait on @milen-sage 's input before making any changes.

Reply:
We always assume file manifests are uploaded in addition to any other options. But I still like calling the manifest_record_type as 'file' or 'file_only' explicitly to indicate that only a manifest file would be uploaded. This can be our default setting. If we go with 'file_only' I think having something along the lines of 'table' or 'table_file' and 'table_entities' will do the trick. We will eventually deprecate 'table_entities'. I'll leave the wording of these parameters to you; we don't need to get stuck on that though as long as the naming is somewhat reasonable and the options are documented and work, CLI/API users will figure it out.

Reply:
@milen-sage should we change the default in this PR or in a new one?

I will go with the following names:
[`file_only`, `file_and_entities`, `table_and_file`, `table_file_and_entities`] just so its more readable and since we aren't limited for space right now.